apps/web/pages/api/generate-referral-link.ts

```diff
@@ -16,9 +16,10 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }
 
   try {
-    return await dub.links.get({
+    const { shortLink } = await dub.links.get({
       externalId: `ext_${session.user.id.toString()}`,
     });
+    return res.status(200).json({ shortLink });
```

Comment:
@steven-tey this is the timeout issue. The response was not being sent when we already had a link.

Reply:
ah good catch!